charts/spire/README.md

```diff
@@ -362,8 +362,10 @@ Now you can interact with the Spire agent socket from your own application. The
 | spire-server.nodeSelector | object | `{}` | Select specific nodes to run on (currently only amd64 is supported by Tornjak) |
 | spire-server.notifier.k8sbundle.namespace | string | `""` | Namespace to push the bundle into, if blank will default to SPIRE Server namespace |
 | spire-server.persistence.accessMode | string | `"ReadWriteOnce"` |  |
+| spire-server.persistence.hostPath | string | `""` | Which path to use on the host when type = hostPath |
 | spire-server.persistence.size | string | `"1Gi"` |  |
 | spire-server.persistence.storageClass | string | `nil` |  |
+| spire-server.persistence.type | string | `"pvc"` | What type of volume to use for persistence. Valid options pvc (recommended), hostPath, emptyDir (testing only) |
```

Comment:
We should default to emptyDir.  That's because it is the most secure, and the one type that is more likely to be present in every deployment.  Those that want better can reconfigure it to match their environment.

Reply:
emptydir is only for certain testing cases. It should not be the default. Letting the clusters chosen PVC driver be used by default is the best option. In a lot of cases, the cluster-admin already made the choice of default volume driver.

Reply:
Understood.  Closing this request